internal/server/v1beta1/search.go

```diff
@@ -28,6 +28,7 @@ func (server *APIServer) SearchAssets(ctx context.Context, req *compassv1beta1.S
 		Filters:    filterConfigFromValues(req.GetFilter()),
 		RankBy:     req.GetRankby(),
 		Queries:    req.GetQuery(),
+		Offset:     int(req.GetOffset()),
```

Comment:
Can we please add tests for this change?

Reply:
done